### ISSUE TITLE
Command line option "--config" stopped working

### ISSUE THREAD

**Issue by mmartinbar** [witnet/witnet-rust#110]
Running the Witnet node with `RUST_LOG=witnet=trace cargo run node --config=<path_to_config_toml>` used to work but now it doesn't.

Running `RUST_LOG=witnet=trace cargo run node -c=<path_to_config_toml>` still works.

Additionally, help should give more useful information about the --config option :
```
>  RUST_LOG=witnet=trace cargo run node --help                                                                                                             
witnet-node 0.1.0
Witnet Foundation <info@witnet.foundation>
Run the Witnet server

USAGE:
    witnet node [OPTIONS]

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
    -c <config>
```

**Comment by csmoe** [comment]
oops, @mmartinbar add a `long` metaitem here, then it'll works.
 `#[structopt(name = "config", long = "config", short = "c")]`:
https://github.com/witnet/witnet-rust/blob/68c32aa2a6047b137888943fd82bbae1eb543e51/src/cli.rs#L37

**Comment by aesedepece** [comment]
Take this chance to put a friendly description for the config option,
something like "path to configuration file"

**Comment by aesedepece** [comment]
Merged in 1466e21e0f5087ca042b7e5e259f938a0e29628b (#111).
